backend/app.py

```diff
@@ -740,6 +786,23 @@ def deleteReport(report_id):
         return jsonify({"error": "An unexpected error occurred. Please try again later."}), 500
 
 
+
+# Get report type "curation" and "companySummarization"
+@app.route("/api/reports/curation/all", methods=["GET"])
```

Comment:
@ramirezmorac2 The path should be one of the following options:
1. `/api/reports` with a URL parameter `type`, which can be either "summarization"(companySummarization) or "curations". This approach allows the same endpoint to handle both types.
2. `/api/reports/summarizations` and `/api/reports/curations` as separate endpoints, where each returns a specific type of report.

This flexibility is necessary because we should be able to pull all the reports, or filter them to only return summarizations(companySummarization) or curations.